dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -237,4 +278,107 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
       out.add(new JdbcSelectTask(file, query).withHeaderClass(Header.class));
     }
   }
+
+  private String getOverrideableQuery(@Nullable String overrideQuery, @Nonnull String defaultSql) {
+    return overrideQuery != null ? overrideQuery : defaultSql;
+  }
+
+  private List<AbstractJdbcTask<Summary>> createAssessmentTasks(ConnectorArguments arguments) {
```

Comment:
Sorry that I wasn't thorough in my previous review. I think for these tables with any time-related columns, we need to allow interval-based extraction. See https://github.com/google/dwh-migration-tools/blob/8c4f10666b7f790321cc2eb1cbab6f9cc9f9b388/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftRawLogsConnector.java#L165-L176